lib/filesystem/include/filesystem/object_status.hpp

```diff
@@ -48,17 +49,20 @@ namespace irods::experimental::filesystem
 
         auto type() const noexcept -> object_type { return type_; }
         auto permissions() const noexcept -> const std::vector<entity_permission>& { return perms_; }
+        auto is_inheritance_enabled() const noexcept -> bool { return inheritance_; }
```

Comment:
@trel @alanking Right now, the fsys library treats the inheritance flag as a boolean. How do we feel about that?

Would it be better to store the value from the catalog and leave the interpretation of it on the developer? We could provide utility functions for translation as well.

This is just me thinking about alternatives to what is presented.

Reply:
well, inheritance *is* a boolean, as far as I know... a collection has it turned on or not.

Reply:
Cool. I think the implementation is as it should be then.